packages/manager/modules/nutanix/src/service.js

```diff
@@ -0,0 +1,125 @@
+import map from 'lodash/map';
```

Comment:
This file is repeated, Ill remove this service in dashboard once another PR is merged to `feat/nutanix`.